packages/api-client-core/src/operationBuilders.ts

```diff
@@ -368,6 +369,44 @@ export const enqueueActionOperation = (
   });
 };
 
+export const enqueueShopifyGraphqlOperation = (
```

Comment:
we aren't going to support `api-client-core` any more so we only need to do this in the mono-repo

see https://github.com/gadget-inc/gadget/blob/main/internal-docs/generated-api-clients.md for more info